### DISCUSSION TITLE
Tip to format a large SDXC card in Fat32 under Windows

### DISCUSSION THREAD

**Discussion by hilsonp** [jomjol/AI-on-the-edge-device#2409] [Q&A]
It is quite difficult to get decent MicroSD card smaller than 64GB those days.
Here is how I formatted a 64GB SDCX card into a 32GB Fat32 under Windows 10:
```
diskpart
list disk
select disk 1
clean
create partition primary size=30000
list partition
select partition 1
active
format quick fs=fat32
assign
exit
```

This would be a nice addition in the FAQ I think ;-)

**Comment by caco3** [comment]
well, at least in switzerland you can get plenty:
https://www.digitec.ch/de/search?filter=n_460%3D32000000000_13&is=micro+sd&q=micro+sd+karte
